llvm/unittests/CGData/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 set(LLVM_LINK_COMPONENTS
   ${LLVM_TARGETS_TO_BUILD}
+  CGData
   CodeGen
-  CodeGenData
   Core
   Support
   )
```

Comment:
Since the directory got renamed, shouldn't the exe be called `CGDataTests` instead of `CodeGenDataTests` too?

Reply:
Yeah. Thanks for catching this! Will follow this up soon.

Reply:
https://github.com/llvm/llvm-project/pull/105463